opencti-platform/opencti-front/src/private/components/events/incidents/IncidentCard.js

```diff
@@ -1,259 +0,0 @@
-    display: 'inline-block',
-    height: '.8em',
-    backgroundColor: theme.palette.grey[800],
-  },
-  placeholder: {
-    display: 'inline-block',
-    height: '1em',
-    backgroundColor: theme.palette.grey[700],
-  },
-});
-
-class IncidentCardComponent extends Component {
-  render() {
-    const { t, fsd, classes, node, onLabelClick } = this.props;
-    return (
-      <Card classes={{ root: classes.card }} variant="outlined">
-        <CardActionArea
-          classes={{ root: classes.area }}
-          component={Link}
-          to={`/dashboard/events/incidents/${node.id}`}
-        >
-          <CardHeader
-            classes={{ root: classes.header }}
-            avatar={
-              <Avatar className={classes.avatar}>{node.name.charAt(0)}</Avatar>
-            }
-            title={node.name}
-            subheader={`${t('Updated the')} ${fsd(node.modified)}`}
-            action={
-              <Fire
-                className={classes.icon}
-                style={{ color: itemColor('Incident') }}
-              />
-            }
-          />
-          <CardContent className={classes.content}>
-            <div className={classes.description}>
-              <Markdown
-                remarkPlugins={[remarkGfm, remarkParse]}
-                parserOptions={{ commonmark: true }}
-                disallowedTypes={['link', 'linkReference']}
-                unwrapDisallowed={true}
-              >
-                {node.description}
-              </Markdown>
-            </div>
-            <div className={classes.objectLabel}>
-              <StixCoreObjectLabels
-                labels={node.objectLabel}
-                onClick={onLabelClick.bind(this)}
-              />
-            </div>
-          </CardContent>
-        </CardActionArea>
-      </Card>
-    );
-  }
-}
-
-IncidentCardComponent.propTypes = {
-  node: PropTypes.object,
-  classes: PropTypes.object,
-  t: PropTypes.func,
-  fsd: PropTypes.func,
-  onLabelClick: PropTypes.func,
-};
-
-const IncidentCardFragment = createFragmentContainer(IncidentCardComponent, {
-  node: graphql`
-    fragment IncidentCard_node on Incident {
-      id
-      name
-      description
-      created
-      modified
-      objectMarking {
-        edges {
-          node {
-            id
-            definition_type
-            definition
-            x_opencti_order
-            x_opencti_color
-          }
-        }
-      }
-      objectLabel {
-        edges {
-          node {
-            id
-            value
-            color
-          }
-        }
-      }
-    }
-  `,
-});
-
-export const IncidentCard = compose(
-  inject18n,
-  withStyles(styles),
-)(IncidentCardFragment);
-
-class IncidentCardDummyComponent extends Component {
-  render() {
-    const { classes } = this.props;
-    return (
-      <Card classes={{ root: classes.cardDummy }} variant="outlined">
-        <CardActionArea classes={{ root: classes.area }}>
-          <CardHeader
```

Comment:
Why the IncidentCardDummyComponent in the new file incidentCard.tsx is not the same as here ?